pkg/validation/config.go

```diff
@@ -24,6 +24,7 @@ type Validator struct {
 
 // NewValidator creates an object that optimizes bulk validations.
 func NewValidator() Validator {
+	// TODO: Changes here?
```

Comment:
If I understand it correctly, when we move cluster profiles from code in `ci-tools` into a config in the `release` repo, so we will need to do some changes in  `pkg/validation/config.go`